src/core/Akka.Remote/Configuration/Remote.conf

```diff
@@ -553,6 +553,18 @@ akka {
           store-location = "current-user"
         }
         suppress-validation = false
+
+        # When enabled, requires mutual TLS authentication where both client and server
+        # must present valid certificates with accessible private keys during the TLS handshake.
+        # This provides defense-in-depth security by ensuring symmetric authentication.
+        #
+        # When disabled, only server-side authentication is performed, which is
+        # sufficient when combined with the startup certificate validation that prevents
+        # servers from starting with inaccessible private keys.
+        #
+        # Set to false only if your environment cannot support client certificate authentication.
+        # Default: true (secure by default)
+        require-mutual-authentication = true
```

Comment:
New mutual TLS requirement - this is technically a breaking behavioral change, but it closes a real security vulnerability so it has to be done.